mlflow/server/spans_router.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+from mlflow.server.handlers import STATIC_PREFIX_ENV_VAR
+
+prefix = os.environ.get(STATIC_PREFIX_ENV_VAR, "")
+router = APIRouter(prefix=f"{prefix}/api/2.0")
+
+
+class LightWeightSpan(BaseModel):
+    trace_id: str
+    span_id: int
+    name: str
+    span_type: str
+    start_time_ms: int
+    end_time_ms: int
+    duration_ms: int
+    status: str
```

Comment:
`SpanInfo`?

Reply:
you mean rename this class to `SpanInfo`?